homeassistant/components/stookwijzer/const.py

```diff
@@ -0,0 +1,17 @@
+"""Constants for the Stookwijzer integration."""
+import logging
+from typing import Final
+
+from homeassistant.backports.enum import StrEnum
+
+DOMAIN: Final = "stookwijzer"
+TRANSLATION_KEY: Final = "stookwijzer"
```

Comment:
This is only used once, in a static place. We don't need to declare a constant for it.

```suggestion
```